server/controllers/userControllers.js

```diff
@@ -52,18 +60,29 @@ exports.login = (req, res) => {
       error: errors.array()[0].msg,
     });
   }
-  const loginUser = userServices.loginUser(req.body);
-  if (loginUser === 'NO USER' || loginUser === 'Invalid password') {
-    return res.send({
-      status: 400,
-      error: 'Invalid email or password',
+  db.query('SELECT * FROM users WHERE email = $1', [email], (err, user) => {
```

Comment:
Also handle the `err` and return the appropriate response.

Reply:
i have handles the errors as requested